src/Hooks/useJumpAI.ts

```diff
@@ -0,0 +1,23 @@
+import { jumpCallbacksOpened } from '../types/GameTypes';
+
+type Props = jumpCallbacksOpened & {
+  aiValue: number;
+  isGameSessionStarted: boolean;
+  paused: boolean;
+  isCameraEnabled: boolean;
+};
+
+export const useJumpAi = ({
+  handleBack,
+  handleJump,
+  aiValue,
+  isGameSessionStarted,
+  paused,
+  isCameraEnabled,
+}: Props): void => {
+  if (!handleJump || !handleBack || !isGameSessionStarted || paused || !isCameraEnabled) return;
+  if (aiValue >= -3) handleJump();
+  else handleBack();
+
+  if (aiValue <= -90) handleBack();
+};
```

Comment:
custom hook that is function that use another hooks. Here I don't see any hooks.

Make useEffect and move to Game.tsx.